### DISCUSSION TITLE
Pairing MIMIC-IV with MIMIC-IV-ECHO

### DISCUSSION THREAD

**Discussion by rthapa84** [MIT-LCP/mimic-code#1821] [MIMIC-IV]
Hi, thank you so much for providing such amazing resources. I am currently downloading the MIMIC-IV-ECHO dataset. On the webpage, it mentions that approximately 12% of cardiologist reports are linked to the echo data. I was wondering, is there a timeline for when more linked reports might become available? Also, if I were to attempt the pairing myself, do you have any recommendations on the best way to go about it?

**Comment by alistairewj** [comment]
No timeline but we are actively working on it!
